docs/proposals/ruler-ha.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Ruler HA"
+linkTitle: "Ruler HA"
+weight: 1
+slug: ruler-ha
+---
+
+- Author: [Soon-Ping Phang](https://github.com/soonping-amzn)
+- Date: Jube 2022
+- Status: Proposed
+---
+
+## Introduction
+
+This proposal consolidates multiple existing PRs from the AWS team working on this feature, as well as future work needed to complete support.  The hope is that a more holistic view will make for more productive discussion and review of the individual changes, as well as provide better tracking of overall progress.
+
+The original issue is [#4435](https://github.com/cortexproject/cortex/issues/4435).
+
+## Problem
+
+Rulers in Cortex currently run with a replication factor of 1, wherein each RuleGroup is assigned to exactly 1 ruler.  This lack of redundancy creates the following risks:
+
+- Rule group evaluation
+  - Missed evaluations due to a ruler outage, possibly caused by a deployment, noisy neighbour, hardware failure, etc.
+  - Missed evaluations due to a ruler brownout due to other tenant rule groups sharing the same ruler (noisy neighbour)
```

Comment:
maybe not within the scope of this proposal but is it possible to detect the impact some how? 

for example, when all the rules for a single tenant is impacted, do we have a way to find out what the impact was?